packages/labs/ssr/src/lib/render-value.ts

```diff
@@ -784,6 +902,25 @@ And the inner template was:
           op.ctor,
           op.staticAttributes
         );
+        if (instance.element) {
+          // In the case the renderer has created an instance, we want to set
+          // the event target parent and the host of the element. Our
+          // EventTarget polyfill uses these values to calculate the
+          // composedPath of a dispatched event.
+          // Note that the event target parent is either the unnamed/named slot
+          // in the parent event target if it exists or the parent event target
+          // if no matching slot exists.
+          const eventTarget = getLast(
+            renderInfo.eventTargetStack
+          ) as HTMLElementWithEventMeta;
+          const slotName = getLast(renderInfo.slotStack);
+          (instance.element as HTMLElementWithEventMeta).__eventTargetParent =
```

Comment:
Would this mean that in the case the elements are not slotted, a sibling's parent node would be incorrectly set to its `previousElementSibling`?

Reply:
Sorry, I'm not sure I fully understand what you mean? Could you elaborate?

Reply:
it will default to eventTarget, which is the result of `getLast(renderInfo.eventTargetStack)` - can that end up being the previous element sibling?

Reply:
Ah, I think I understand now what is meant.
No, because the entry is removed with the closing of the custom element or slot element down below.